src/debugging/netcore/NetCoreDebugHelper.ts

```diff
@@ -228,6 +285,39 @@ export class NetCoreDebugHelper implements DebugHelper {
 
         return pathNormalize(result, platformOS);
     }
+
+    private async InstallDebuggerOnContainer(containerName: string): Promise<string> {
+        const debuggerPath: string = '/remote_debugger';
+        const installUnZip: string = '/bin/sh -c "apt-get update && apt-get install unzip"';
+        const installDebugger: string = `/bin/sh -c "curl -sSL https://aka.ms/getvsdbgsh | /bin/sh /dev/stdin -v latest -l ${debuggerPath}"`;
+
+        const outputManager = new DefaultOutputManager(ext.outputChannel);
+        const dockerClient = new CliDockerClient(new ChildProcessProvider());
+        const options = { interactive: true };
+
+        await outputManager.performOperation(

```

Comment:
Should we warn the user that we're modifying the running container (by installing additional packages and the debugger)?  Since we're scaffolding the launch configuration for this attach process, making it immediately available, the user may not understand the implications of using it.

Reply:
Are you suggesting just a warning on the output window or some kind of popup that user has to confirm?
Just showing a warning doesn't offer an option to stop the installation, but the pop-up will be annoying.

Reply:
I'd suggest a pop-up on the first attempt to attach to a container (i.e. if we detect that the debugger is not yet installed), to minimize the pop-up, perhaps with a "don't prompt again" option for users that find even that annoying.  While containers are assumed to be ephemeral, they're also assumed to be (relatively) identical.  Since we're not creating the container (so there's no notion of ownership of the container), and we're actively changing that running container, and it'll still be different when we're done debugging, it seems like we should make users aware of that fact.

Reply:
Now showing an information message to user asking to allow or not.